Project.toml

```
@@ -17,6 +18,7 @@ Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 PlotsExt = "Plots"
 
 [compat]
+LaTeXStrings = "1.3.1"
```

```suggestion
LaTeXStrings = "1"
```
Sorry, I wasn't clear enough above - I had in mind just adding the major version number (unless you know that you need some features introduced in v1.3).